packages/wallets/src/components/WalletsAddMoreCardBanner/WalletsAddMoreCardBanner.scss

```diff
@@ -20,6 +20,7 @@
             justify-content: space-between;
         }
         &-button {
+            z-index: 10;
```

Comment:
**nit**: Is this a good practice to use `z-index`? Can we think of an alternative to it? (Just a suggestion)

Reply:
It is because the gradient shine use absolute position and it makes the button unclickable. Please give suggestion if you have another solution 🙇